forch/faucetizer.py

```diff
@@ -105,7 +103,9 @@ def _faucetize(self):
 
         self._behavioral_faucet_config = behavioral_faucet_config
 
-    def _write_behavioral_config(self):
+    def _flush_behavioral_config(self, force=False):
+        if not force and self._config.faucetize_interval_sec:
+            return
```

Comment:
this method still overlaps with write_behavioral_config, so should be consolidated. Also write_behavioral_config should be class method, not a static one.  The objective here is to consolidate as much common stuff in one place as possible -- so the yaml.dump method should only in one place, and then have other functions that provide other functionality on top of that. In the long run, this minimizes mistakes since there's only one place something happens, so you don't need to chase down and/or edit multiple instances of something.

Reply:
OK, moved both write_behavioral_config and update_structural_config into Faucetizer as class methods.